src/App.jsx

```diff
@@ -1,3 +1,70 @@
-export default function App() {
-  return <h1 className="text-3xl font-bold underline bg-blue-500">React exercises</h1>;
-}
+import { useState, useEffect } from "react";
+import CardComponent from "./components/CardComponent";
+
+const App = () => {
+  const [count, setCount] = useState(0);
+  const [usersData, setUsersData] = useState([]);
+
+  useEffect(() => {
+    async function getData() {
+      try {
+        const res = await fetch(
+          "https://random-data-api.com/api/users/random_user?size=10"
+        );
+        const data = await res.json();
+        const newArray = Array.from(data, ({ avatar: image, first_name: fname, last_name: lname, employment }) => ({
+          image,
+          fname,
+          lname,
+          employment,
+        }));
+        setUsersData(newArray);
+      } catch (error) {
+        console.error(error);
+      }
```

Comment:
You can remove this console.error(error) instead stores the error in its state as it may be a good way of processing an error instead of logging it on the console.